CONTRIBUTING.md

```diff
@@ -228,11 +228,13 @@ This will do a symbolic link from the global `node_modules` version to point to
 For testing, you will want to generate an application, and there is a specific issue here: for each application, JHipster installs a local version of itself. This is made to enable several applications to each use a specific JHipster version (application A uses JHipster 3.1.0, and application B uses JHipster 3.2.0).
 
 To overcome this you need to run `npm link generator-jhipster` on the generated project folder as well, so that the local version has a symbolic link to the development version of JHipster.
+Also add the option `--skip-jhipster-dependencies` to generate the application ignoring the JHipster dependencies (otherwise a released version with be installed each time npm install/ci is called). You can later on re-add the dependency with the command `jhipster --no-skip-jhipster-dependencies`.
```

Comment:
with be -> will be

Reply:
the error is also here:  https://github.com/jhipster/jhipster.github.io/pull/998/files#diff-582158a3a484757228416d0a6de1af7eR223

Reply:
I'm fixing it too

Reply:
PR : https://github.com/jhipster/jhipster.github.io/pull/1018

Reply:
thanks :)